pandas/tests/io/test_s3.py

```diff
@@ -6,3 +10,13 @@ class TestS3URL(object):
     def test_is_s3_url(self):
         assert is_s3_url("s3://pandas/somethingelse.com")
         assert not is_s3_url("s4://pandas/somethingelse.com")
+
+
+def test_streaming_s3_objects():
+    data = [
```

Comment:
Then here you can do `botocore_response = pytest.importorskip('botocore.response', minversion='1.10.47')`

Reply:
Can we do this when the version pertains to the base module, not the imported class? I just tried it in a dummy file and all I got was `Skipped: module 'botocore.response' has __version__ None, required is: '1.10.47'`. So should I do something like this?

```
import pytest

def my_test_fn():
    pytest.importorskip('botocore', minversion='1.10.47')
    from botocore.response import StreamingBody
    # test
```